modules/psap-special-resource-operator-metrics.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="special-resource-operator-metrics_{context}"]
+= Prometeus Special Resource Operator metrics
+
+
+The Special Resource Operator (SRO) exposes the following Prometheus metrics through the `metrics` service:
+
+|===
+|Metric Name |Description
+
+|`sro_used_nodes`
+|Returns the nodes that are running pods created by a SRO custom resource (CR). This metric is available for `DaemonSet` and `Deployment` objects only. 
+
+|`sro_kind_completed_info`
+|Represents whether a `kind` of SRO CR object has been successfully uploaded in the cluster (value `1`) or not (value `0`). Examples of objects are `DaemonSet`, `Deployment` or `BuildConfig`.
+
+|`sro_states_completed_info`
+|Represents whether the SRO has finished processing a CR successfully (value `1`) or not (value `0`).
```

Comment:
value 0 means that CR has not been processed yet.  Operator will always try to process a CR until successful completion. If there is an error, then a CR will be rescheduled